build.sbt

```diff
@@ -11,11 +11,12 @@ val commonDeps = Seq(
   "org.apache.curator" % "curator-x-discovery" % "2.4.2" % "optional",
   "com.socrata"       %% "socrata-http-jetty"  % "3.0.2" % "optional",
   "com.socrata"       %% "socrata-http-client" % "3.0.2" % "optional",
+  "com.nativelibs4java" %% "scalaxy-loops"     % "0.3.3" % "provided",
   "org.scalacheck"    %% "scalacheck"          % "1.12.2" % "test",
   "org.scalatest"     %% "scalatest"           % "2.2.1" % "test",
   "com.rojoma"        %% "simple-arm"          % "[1.2.0,2.0.0)",
   "com.rojoma"        %% "simple-arm-v2"       % "2.0.0" % "optional",
```

Comment:
Binary compatibility: scala lang 2.10.0 is evicted and updated to 2.10.2; is it worthwhile to update in this project's build?